routes/payment.js

```diff
@@ -66,13 +64,13 @@ router.post('/create-session', async (req, res) => {
     const stripeId = user.stripeId;
     const guestFee = reservationDetails.guestFee;
     const listingPrice = listingDetails.price;
-
+    const centsToDollars = 100; 
 
     // Total price of the listing including guest fee. Host fee + our profit is taken from this. 
-    const listingTotal = (listingPrice * days * 100) + (guestFee * 100);
+    const listingTotal = (listingPrice * days * centsToDollars) + (guestFee * centsToDollars);
 
     // Price we make - Host Fee (1%) + Guest Fee (10%)
-    const applicationFee = (listingPrice * days * 100) * 0.01 + (guestFee * 100)
+    const applicationFee = (listingPrice * days * centsToDollars) * 0.01 + (guestFee * centsToDollars)
```

Comment:
The 0.01 should also be changed to a constant. We should move all global variables to the top of the file as well